src/bootstrap/src/utils/job.rs

```diff
@@ -6,8 +6,10 @@ pub unsafe fn setup(_build: &mut crate::Build) {}
 
 #[cfg(all(unix, not(target_os = "haiku")))]
 pub unsafe fn setup(build: &mut crate::Build) {
-    if build.config.low_priority {
-        libc::setpriority(libc::PRIO_PGRP as _, 0, 10);
+    unsafe {
+        if build.config.low_priority {
+            libc::setpriority(libc::PRIO_PGRP as _, 0, 10);
```

Comment:
Only the setpriority call needs to be unsafe, right?

Reply:
Good point, this was automated by `cargo fix`, I'll reduce the scope.